src/uikit.cs

```diff
@@ -21156,4 +21200,167 @@ interface UIKey : NSCopying, NSCoding {
 		UIKeyboardHidUsage KeyCode { get; }
 	}
 
+	[NoWatch, NoTV, iOS (13,4)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UIPointerInteraction : UIInteraction {
+
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		IUIPointerInteractionDelegate Delegate { get; }
```

Comment:
^ should expose `WeakDelegate`

Reply:
I intentionally removed it since there is not setter